doc/new-book/configuration-security.adoc

```diff
@@ -412,3 +414,385 @@ listener {
++
+--
+The connection limits apply to all users that are connected to the vhost. These limits control the number of users that can be connected simultaneously to the vhost.
+
+[options="nowrap",subs="+quotes"]
+----
+vhost = {
+    id: _HOSTNAME_
+    maxConnections: _NUMBER_OF_CONNECTIONS_
+    maxConnectionsPerUser: _NUMBER_OF_CONNECTIONS_
+    maxConnectionsPerHost: _NUMBER_OF_CONNECTIONS_
+    allowUnknownUser: true | false
+    ...
+}
+----
+
+`id`::
+The host name of the vhost. This vhost policy will be applied to any client connection that is directed to the hostname that you specify.
+
+`maxConnections`::
+The global maximum number of concurrent client connections allowed for this vhost. The default is `65535`.
+
+`maxConnectionsPerUser`::
+The maximum number of concurrent client connections allowed for any user. The default is `65535`.
+
+`maxConnectionsPerHost`::
+The maximum number of concurrent client connections allowed for any remote host (the host from which the client is connecting). The default is `65535`. 
+
+`allowUnknownUser`::
+Whether unknown users (users who are not members of a defined user group) are allowed to connect to the vhost. Unknown users are assigned to the `$default` user group and receive `$default` settings. The default is `false`, which means that unknown users are not allowed.
+--
+
+. In the `vhost` section, beneath the connection settings that you added, add the necessary user groups.
++
+--
+A user group defines what messaging resources the members of the group are allowed to access.
+
+[options="nowrap",subs="+quotes"]
+----
+vhost {
+    ...
+    groups: {
+        _USER_GROUP_: {
+            users: _USERNAME_, ...
+            remoteHosts: _HOSTNAME_ | _IP_ADDRESS_ | _IP_ADDRESS_RANGE_
+            sources: _SOURCE_ADDRESS_, ...
+            targets: _TARGET_ADDRESS_, ...
+        },
+        ...
+    }
+}
+----
+
+`users`::
+A list of authenticated users for this user group. Use commas to separate multiple users. A user may belong to only one vhost user group.
++
+.Specifying Users
+====
+[options="nowrap"]
+----
+users: user1, user2, user3
+----
+====
+
+`remoteHosts`::
+A list of remote hosts from which the users may connect. A host can be a hostname, IP address, or IP address range. Use commas to separate multiple hosts. To allow access from all remote hosts, specify a wildcard `*`. To deny access from all remote hosts, leave this attribute blank.
++
+.Setting Remote Hosts
+====
+In this example, users in this user group can only connect to the vhost from localhost:
+
+[options="nowrap"]
+----
+remoteHosts: 127.0.0.1, ::1
+----
+====
+
+`sources`::
+A list of source addresses from which users in this group may receive messages. To specify multiple addresses, separate the addresses with either a comma or a space. If you do not specify any addresses, users in this group are not allowed to receive messages from any addresses.
```

Comment:
Maybe call them 'AMQP source addresses' to signal to the reader that it is not a network address but a protocol thing. Same for targets.